modules/ROOT/pages/functions/string.adoc

```diff
@@ -148,6 +148,163 @@ RETURN ltrim('   hello')
 
 ======
 
+
+
+[[functions-normalize]]
+== normalize()
+
+`normalize()` returns the given `STRING` normalized using the `NFC` normalization form.
+
+The `normalize()` function is useful for converting `STRING` values into comparable forms.
+When comparing two `STRING` values, it is their Unicode codepoints that are compared.
+In Unicode, a codepoint for a character that looks the same may actually be represented by two, or more, different codepoints.
+For example, the character `<` can be represented as `\uFE64` or `\u003C`. Visually, the character may look the same,
+but if compared, Cypher will return false as `\uFE64` does not equal `\u003C`. Using the `normalize()` function one can
+normalize the codepoint `\uFE64` to `\u003C`, creating a single codepoint representation, allowing them to be successfully compared.
+
+*Syntax:*
+
+[source, syntax, role="noheader"]
+----
+normalize(input)
+----
+
+*Returns:*
+
+|===
+
+| `STRING`
+
+|===
+
+*Arguments:*
+
+[options="header"]
+|===
+| Name | Description
+
+| `input`
+| An expression that returns a `STRING`.
+
+|===
+
+*Considerations:*
+
+|===
+
+| `normalize(null)` returns `null`.
+
+|===
+
+
+.+normalize()+
+======
+
+.Query
+[source, cypher, indent=0]
+----
+RETURN normalize('\u212B') = '\u00C5' AS result
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+|===
+
+| +result+
+| +true+
+1+d|Rows: 1
+
+|===
+
+======
+
+
+[[functions-normalize-with-normal-form]]
+== normalize(), with specified normal form
+
+`normalize()` returns the given `STRING` normalized using the specified normalization form.
+The normalization form can be of type `NFC`, `NFD`, `NFKC` or `NFKD`.
+
+There are two main types of normalization forms. One is based on the concept of canonical equivalence, and the other is based on compatibility.
+
+The two forms `NFC` (default) and `NFD` are forms of canonical equivalence. This means that codepoints which represent the same abstract character will
+be normalized to the same codepoint. The same abstract character means that the character has the same visual appearance and behavior.
+The difference between `NFC` and `NFD` is that `NFC` form will always give the *composed* canonical form, one where combined codes are replaced with the single representation, if possible.
+Whereas, `NFD` gives the *decomposed* form, this is the opposite of composed, converting combined codepoints into the split form if possible.
```

Comment:
What do we mean by "the combined codes"?

```suggestion
*  *Canonical equivalence*: The `NFC` (default) and `NFD` are forms of canonical equivalence.
This means that codepoints that represent the same abstract character will
be normalized to the same codepoint (and have the same appearance and behavior).
The `NFC` form will always give the *composed* canonical form (in which the combined codes are replaced with a single representation, if possible).
The`NFD` form gives the *decomposed* form (the opposite of the composed form, which converts the combined codepoints into a split form if possible).
```

Reply:
So a combining unicode is one that will overlay itself on the previous character, e.g for Å this can be just Å or it can be A + a combining char for the little dot. So the codes "combine" together to form one character (visually)